php-exercise-1/Model/helloModel.php

```diff
@@ -0,0 +1,6 @@
+<?php
+class HelloModel {
+    public function getMessage() {
```

Comment:
Please refer to PSR-1 and PSR-12 for best practices when writing your PHP scripts.

Reply:
I have edited the scripts to adhere to PSR-1 and PSR-12